antora-ui/src/css/antora-ui.css

```diff
@@ -80,6 +87,14 @@ html:not(.is-clipped--nav) main.article {
   }
 }
 
+@media screen and (min-width: 1280px) {
+  main.article {
+    flex: 1;
+    max-width: calc(100% - 18rem);
```

Comment:
I wonder how this would look full screen on Vinnie's monitor.

Reply:
![image](https://github.com/user-attachments/assets/182b7c23-11a4-4d1f-992a-074e00b5c754)